rebasehelper/tests/conftest.py

```diff
@@ -59,3 +62,20 @@ def pytest_collection_modifyitems(items):
             item.add_marker(pytest.mark.functional)
         else:
             item.add_marker(pytest.mark.standard)
+
+
+@pytest.fixture
+def mock_spec_object_generator(monkeypatch):
```

Comment:
I don't like the name. The fixture doesn't return a generator. Also, I'd rather see `mocked` instead of `mock`.
Also, wouldn't it be more convenient to not to return a "decorator", but `SpecFile` instance with attributes set up via traditional method arguments (that can be directly parametrized)?

Reply:
If we want to keep this as a pytest fixture, I don't think that there is a way to directly return SpecFile while using arguments to influence the `SpecFile`. It could be changed to a normal function.. Wouldn't it have any downsides when using it in our tests?

Reply:
What makes you think so? [This fixture](https://github.com/rebase-helper/rebase-helper/blob/master/rebasehelper/tests/test_config.py#L38) does something like that.